packages/shared/utils/is.ts

```diff
@@ -23,3 +25,11 @@ export const rand = (min: number, max: number) => {
   return Math.floor(Math.random() * (max - min + 1)) + min
 }
 export const isIOS = /* #__PURE__ */ isClient && window?.navigator?.userAgent && /iP(ad|hone|od)/.test(window.navigator.userAgent)
+
+export function isSup(callback: () => boolean) {
+  const isSupported = ref(callback())
+  nextTick(() => {
```

Comment:
```suggestion
  onMounted(() => {
```

And we should move this to `@vueuse/core` instead of shared.